### PR TITLE
correct dimension input for non-square matrix

### PR DESCRIPTION
**Pull request by aTrotier**
Related to #143

### CODE REVIEW

**Comment by tknopp** [comment]
Looks good.

One question though: Why did you add dependencies to `MRIReco` in [Project.toml](https://github.com/MagneticResonanceImaging/MRIReco.jl/pull/144#diff-72ed386c2a0cd1d23c0968297e70023ed98c22490d146dd89fc91f48369bad4d)? These should not be necessary.

**Comment by aTrotier** [comment]
It was added when building the documentation offline.

I still have some errors during CI for the documentation. I think the subpackages are not dev ?

File: docs/lit/examples/02-extractKspace.jl [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+undersample the data and create back and AcquisitionData structure that can be used for reconstruction.
+
+This possibility can be easily combined with the [BartIO](https://github.com/MagneticResonanceImaging/BartIO.jl) package to reconstruct the kspace.
+=#
+
+# ## Setup
+using CairoMakie
+using ImageUtils: shepp_logan
+using MRIReco, MRISimulation
+
```
[the remark this answers is not in the document]
Comment (JeffFessler) [reply docs/lit/examples/02-extractKspace.jl:19]: i added InteractiveUtils to docs/Project.toml
if you commit this then it will fix the issue

**Comment by aTrotier** [comment]
@tknopp I think it is ready to merge :)

**Comment by tknopp** [comment]
I still don't understand why dependencies have been added to Project.toml:
https://github.com/MagneticResonanceImaging/MRIReco.jl/pull/144/files#diff-72ed386c2a0cd1d23c0968297e70023ed98c22490d146dd89fc91f48369bad4d
The dependencies are not used within `MRIReco`.

**Comment by JeffFessler** [comment]
I bet those dependencies belong in `docs/Project.toml`...

**Pull request merged**